src/GraphQL/Types/Collections/SchemaTypes.cs

```diff
@@ -791,7 +791,12 @@ private object GetGraphType(Type clrType, bool input, IEnumerable<IGraphTypeMapp
             var ret = GetGraphTypeFromClrType(clrType, input, typeMappings);
```

Comment:
Exception will be thrown here for `IGraphType` CLR types; additional code in this method not required.

Reply:
Sorry, thinking of wrong method.

Reply:
When I wrote the code, I also got confused in them.